packages/inter-protocol/src/vaultFactory/params.js

```diff
@@ -210,9 +210,9 @@ export const provideVaultParamManagers = (
 
   // restore from baggage, unless `managerParamOverrides` overrides.
   for (const [brand, args] of managerArgs.entries()) {
-    const newInitial = Object.values(managerParamOverrides).find(
-      e => e.brand === brand,
-    );
+    const newInitial = managerParamOverrides
+      ? Object.values(managerParamOverrides).find(e => e.brand === brand)
+      : undefined;
```

Comment:
isn't this what `find()` returns when nothing matches the predicate?

Reply:
`Object.values()` is unhappy when `managerParamOverrides` is `undefined`.

Reply:
ah. I'm surprised it's ever undefined because the function signature says it's `Record<string, VaultManagerParamOverrides>`. If optional please give it a default of `{}` instead.